packages/core/src/errors.ts

```diff
@@ -1,13 +1,13 @@
 export class FileNotFoundError extends Error {
 	constructor(readonly filePath: string) {
-		super(`File not found: ${filePath}`);
+		super('File not found', { cause: { filePath } });
```

Comment:
small issue: the debug log does not show the `cause` field currently, so the error will be missing some vital details there...